### PR TITLE
Rename org to projectgroup

### PR DESCRIPTION
**Pull request by iAKM**
Renamed all the occurrences of organization to project group

### CODE REVIEW

File: src/python_ms_core/core/auth/models/permission_request.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def get_search_params(self):
         affirmative = 'true' if self._should_satisfy_all else 'false'
         params = {
             'userId': self._user_id,
-            'agencyId': self._org_id,
+            'agencyId': self.project_group_id,
```
Comment (susrisha) [inline src/python_ms_core/core/auth/models/permission_request.py]: Is this the same parameter `agencyId` ?

File: tests/unit_tests/test_auth/models/test_permission_request.py [hunk 2]
```diff
@@ -6,16 +6,16 @@ class TestPermissionRequest(unittest.TestCase):
...
         should_satisfy_all = True
 
-        request = PermissionRequest(user_id, org_id, permissions, should_satisfy_all)
+        request = PermissionRequest(user_id, project_group_id, permissions, should_satisfy_all)
         search_params = request.get_search_params()
 
         expected_params = {
             'userId': user_id,
-            'agencyId': org_id,
+            'agencyId': project_group_id,
```
Comment (susrisha) [inline tests/unit_tests/test_auth/models/test_permission_request.py]: Is this the same parameter `agencyId` ?

**Comment by susrisha** [comment]
Couple of things to be done:
1. Check if the unit tests are successful
2. Change the version of core package .

**Review by susrisha — CHANGES_REQUESTED** [review]
Will have to wait till the unit tests and integration tests are successful

File: src/python_ms_core/core/auth/models/permission_request.py (hunk 1 above)
Comment (MashB) [reply src/python_ms_core/core/auth/models/permission_request.py]: AgencyId also to be replaced with projectGroupId

**Review by susrisha — APPROVED** [review]

**Review by MashB — APPROVED** [review]

**Pull request merged**